geometry_msgs/Pose2D tp_pos
bool visited
